test/nginx_system_test.sh

```diff
@@ -1315,7 +1315,7 @@ start_test Shutting down.
 fire_ab_load
 
 if $USE_VALGRIND; then
-    kill -s quit $VALGRIND_PID
+    kill -s TERM $VALGRIND_PID
```

Comment:
It was not, plus it causes an additional log message that is problematic for the system test. Fixed.